### PR TITLE
First

### PR DESCRIPTION
**Pull request by g4s8**
First element in collection scalar
Part of #13

### CODE REVIEW

**Comment by 0crat** [comment]
@g4s8 I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

File: src/main/java/org/cactoos/list/First.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    public First(final Iterable<T> source) {
+        this.source = source;
+    }
+
+    @Override
+    public T asValue() throws IOException {
+        final Iterator<T> iterator = this.source.iterator();
+        if (iterator.hasNext()) {
+            return iterator.next();
+        } else {
```
Comment (yegor256) [inline src/main/java/org/cactoos/list/First.java]: @g4s8 check this out: http://www.yegor256.com/2015/01/21/if-then-throw-else.html

**Comment by yegor256** [comment]
@g4s8 see my comment above

**Comment by g4s8** [comment]
@yegor256 looks logical, I've fixed it

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**